test/old/testdir/test_python3.vim

```diff
@@ -1,9 +1,79 @@
 " Test for python 3 commands.
```

Comment:
This can be cherry-picked but any `test_` functions should be skipped or modified to disable failing cases.